### DISCUSSION TITLE
Utilities: Data Search and Replace, preface field name with field id

### DISCUSSION THREAD

**Discussion by creativearc-paul** [ExpressionEngine/ExpressionEngine#4463] [Feature Requests]
### Suggest an idea for ExpressionEngine

I rarely need to use the 'Search and Replace' feature, but it's very handy when it's there.

It'd be a help to preface the select menu (listing all the field names) with their respective field_id. Sometimes field names are similar, and this would make this replace action a bit more predictable.
<img width="1113" alt="field-id-replace" src="https://github.com/user-attachments/assets/7b3ac578-6074-4929-a702-f70afaec9542">

**Comment by intoeetive** [comment]
https://github.com/ExpressionEngine/ExpressionEngine/pull/4807
